.github/workflows/soak.yml

```diff
@@ -332,5 +332,52 @@ jobs:
       - name: Detect regressions
         run: |
           ./soaks/bin/detect_regressions --capture-dir ${{ github.event.number }}-${{ github.run_attempt }}-captures/ \
-                                         --erratic-soaks syslog_loki,http_to_http_acks \
+                                         --warmup-seconds 30 \
                                          --p-value 0.1
+
+  plot-analysis:
+    name: Plot analysis
+    if: ${{ github.actor != 'dependabot[bot]' }}
+    runs-on: ubuntu-20.04
+    needs:
+      - compute-soak-meta
+      - soak
+
+    steps:
+      - name: Set up Python3
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.10.1"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install scipy==1.7.*
+          pip install pandas==1.3.*
+          pip install seaborn==0.11.*
+          pip install tabulate==0.8.*
```

Comment:
Not knowing much all too much about Python - but I'm assuming if you do `pip install foo==x.x.* bar=x.x.* ...` instead of a command for each of those individually that the package manager would be able to simultaneously download packages

Reply:
Python doing things simultaneously :not_sure_if:

Reply:
Oh yeah, indeed. https://stackoverflow.com/a/9956813